sink/sink

```diff
@@ -339,6 +349,10 @@ class Status(object):
         for reporter in self.reporters:
             try:
                 reporter.push(data)
+            except GithubPushError:
+                stderr.write(b"State not as expected. Possible collision. Aborting.\n")
+                stderr.flush()
+                sys.exit(EXIT_COLLISION)
             except:
```

Comment:
You can change this to `except Exception:`.  `SystemExit` derives from `BaseException`, but not `Exception`, and is the only class in the standard library for which this is true.  That will allow the exit to pass through, but will still catch all of the "real" exceptions.